### PR TITLE
SEO-176062-Aurelia-controls-issue-fixes

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: Aurelia/Maps/Getting-Started.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ You can configure an **Essential Aurelia** Map in simple steps. In this example,
 
 To use ejMap, refer the following libraries in **HTML** page. 
 
-* [jQuery](http://jquery.com/) version  1.10.1 and above,
+* [jQuery](https://jquery.com) version  1.10.1 and above,
```
Comment (Yvone-Atieno) [inline Aurelia/Maps/Getting-Started.md:43]: Revert this change

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Aurelia/Maps/Getting-Started.md (hunk 1 above)
Comment (Edithabich) [reply Aurelia/Maps/Getting-Started.md:43]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**